### DISCUSSION TITLE
date functions supporting decimals?

### DISCUSSION THREAD

**Discussion by amirrajan** [duckdb/duckdb#3252] [Q&A]
Is there a workaround for date functions requiring an integer? For example if I want to add 1.5 days to a date.

**Comment by amirrajan** [comment]
Nevermind. I can floor the value :-)

**Reply by hawkfish** [reply]
I think you have to convert it to the appropriate fraction. We support microseconds, so multiply by 86,400,000,000 in the worst case. And in your example, just * 24 to get hours ought to do it. 

But we really should add `DATE + DOUBLE` for this situation.
